onnxruntime/python/tools/kernel_explorer/kernel_explorer.cc

```diff
@@ -22,6 +23,7 @@ PYBIND11_MODULE(_kernel_explorer, m) {
   InitFastGelu(m);
   InitGemm(m);
   InitSkipLayerNorm(m);
+  InitGemmFastGelu(m);
```

Comment:
Backlog: we should move to distributed kernels profiling register, instead of keeping them centralized. It is not easy to maintain these types of registration and with CUDA is now also supported. @PeixuanZuo you can volunteer implement this if you'd like to.